caas/kubernetes/provider/base_test.go

```diff
@@ -202,14 +201,14 @@ func (s *BaseSuite) setupController(c *gc.C) *gomock.Controller {
 		return "appuuid", nil
 	}
 
-	s.mockNamespaces.EXPECT().Get(gomock.Any(), s.getNamespace(), v1.GetOptions{}).
+	s.mockNamespaces.EXPECT().Get(gomock.Any(), s.getNamespace(), v1.GetOptions{}).Times(2).
```

Comment:
I think just making this .Any() would be fine.

Reply:
I'd prefer asserting more explicitly. And also set to AnyTimes() here might fail the later namespace get assertion statements in other real tests.